src/component/header/index.tsx

```diff
@@ -11,9 +11,11 @@ import {
 } from "component/plasmic/shared/PlasmicHeader";
 import { IUser } from "module/interface/user";
 
+type IsVisible = "mobileSidebar" | "postDetails" | "showModal";
```

Comment:
This is defined in quite a few places. Should probably be extracted into a common file.

Reply:
Sounds good